src/components/PersonLink.tsx

```diff
@@ -0,0 +1,18 @@
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+import { FC } from 'react';
+
+export type Props = {
+  to: string;
+  text: string;
+  sex?: string;
+};
+
+export const PersonLink: FC<Props> = ({ to, text, sex }) => (
+  <NavLink
+    to={`../${to}`}
+    className={sex === 'f' ? classNames('has-text-danger') : ''}
```

Comment:
You can simplify the className assignment using the ternary operator directly inside the string instead of using 'classNames'.

Reply:
I don't now why, but it is only one way when its works correctly